packages/parcel-optimizer-html-inline-css/src/index.ts

```diff
@@ -0,0 +1,43 @@
+import assert from 'assert';
+
+import { sort } from '@compiled/css';
+import { Optimizer } from '@parcel/plugin';
+import posthtml from "posthtml";
+import { insertAt } from "posthtml-insert-at";
+
+export default new Optimizer({
+  async optimize({ contents, map, bundleGraph }) {
+    const styleRules: string[] = [];
+
+    for (const bundle of bundleGraph.getBundles()) {
```

Comment:
AFAIK this will get all the bundles, not just the bundles that this HTML is the entry for - is that expected? (i.e. every HTML file will end up with the same styles in it with this approach - which I know we did discuss, but can't remember where we ended up on whether that was what we wanted)